service/server.py

```diff
@@ -114,33 +109,33 @@ def create_json(address_rec, latest_sale):
         'street': address_rec.thoroughfareName.rstrip(),
         'town': address_rec.postTown.rstrip(),
         'county': address_rec.dependentLocality.rstrip(),
-        'postcode': address_rec.postcode.rstrip(),
+        'postcode': address_rec.postCode.rstrip(),
         'amount': latest_sale.get('amount', None),
         'date': latest_sale.get('date', None),
         'property_type':
             get_property_type(latest_sale.get('property_type', None)),
         'coordinates' : {
-            'latitude': address_rec.positionY,
-            'longitude': address_rec.positionX,
+            'latitude': address_rec.position.y,
```

Comment:
That's the API response. If we change it, the frontend will have to do that, too. Perhaps it would be best to do it in a separate pair of pull requests?